test/test_post_action_verification.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 from unittest.mock import Mock
 
-from build.lib.puma.state_graph.puma_driver import PumaClickException
```

Comment:
I had seen this weird import before, but didn't understand what caused it. I guess the difference between `test`and `test_scripts`?